test_cromwell.sh

```diff
@@ -11,51 +11,50 @@ trap "shutdown" EXIT
 
 set -e
 
-EXIT_CODE=31
+EXIT_CODE=1
```

Comment:
Cuz 31 I left in by accident earlier on when I was making sure I was seeing this particular exit code.

This is the default exit code, it gets flipped to 0 at the very end. The reason is that I have the EXIT trap to clean things up at the end but that trap function can't know the exit code that led to the exit.